internal/binutils/binutils.go

```
@@ -120,6 +121,29 @@ func (bu *Binutils) SetTools(config string) {
 	bu.update(func(r *binrep) { initTools(r, config) })
 }
 
+func ensureObjdumpVersion(objdump string, objdumpFound bool) (string, bool) {
+	if !objdumpFound || runtime.GOOS != "darwin" {
+		return objdump, objdumpFound
+	}
+
+	// Ensure objdump is at least version 9.0 on Mac OS
+	cmdOut, err := exec.Command(objdump, "-version").Output()
+	if err != nil {
+		return objdump, false
+	}
+
+	re := regexp.MustCompile(`.*version (([0-9]*)\.([0-9]*)\.([0-9]*)).*$`)
+	fields := re.FindStringSubmatch(strings.Split(string(cmdOut), "\n")[0])
+	if len(fields) != 5 {
+		return objdump, false
+	}
+
+	if ver, _ := strconv.Atoi(fields[2]); ver < 9 {
```

Don't ignore errors.

---

done.